config/released-configs.schema.json

```diff
@@ -0,0 +1,19 @@
+{
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "title": "Deployment Environments",
+    "description": "A list of deployment targets that are supported",
+    "type": "object",
+    "properties": {
+        "$schema": {
+            "type": "string"
+        },
+        "tags": {
+            "type": "array",
+            "items": {
+                "type": "string"
+            }
+        }
+    },
+    "required": ["tags"],
+    "additionalProperties": false
+}
```

Comment:
Can of worms: we're not thinking we'll ever need to version this? I guess it's for internal consumption only, and we'd just update the `json` if we changed the schema, so no need for backwards compatibility.

Again, this is mostly thinking out loud for my future self.

Reply:
Noted. I have been modifying the schema to fit the json as I go. It's more just for it to be internally consistent if I'm modifying things in the json that fit the schema already.